sklearn/ensemble/tests/test_voting_classifier.py

```diff
@@ -14,11 +16,28 @@
 from sklearn.svm import SVC
 from sklearn.multiclass import OneVsRestClassifier
 
+
 # Load the iris dataset and randomly permute it
 iris = datasets.load_iris()
 X, y = iris.data[:, 1:3], iris.target
 
 
+def test_init_parameters():
+    eclf = VotingClassifier(estimators=[])
+    assert_raises(AttributeError, eclf.fit, X, y)
```

Comment:
Ideally, we should use `assert_raise_message`, to improve robustness and readability.